src/fft_convolve.jl

```diff
@@ -1,139 +1,134 @@
 # fft_convolve.jl
 
 export fft_conv!, fft_conv_adj!
+export fft_conv, fft_conv_adj
 
-import AbstractFFTs
-import FFTW
-using FFTW: plan_fft!, plan_ifft!
-
+"""
+    imfilterz!(plan)
+FFT-based convolution between `plan.img_compl` and kernel `plan.ker_compl` (not centered)
+putting result in `plan.workmat`.
+"""
+function imfilterz!(
+    plan::PlanPSF,
+    )
```

Comment:
```suggestion
function imfilterz!(plan::PlanPSF)
```
if it is short enough then this format is preferable